src/greatwall/gui.py

```diff
@@ -1024,11 +1030,14 @@ def config_selecting_derivation_widgets(self):
                             )
                     )
 
-                selection_button.clicked.connect(
-                    lambda state, selection_idx=idx: self.on_selection_button_click(
-                        selection_idx
+                if idx == 0:
+                    selection_button.clicked.connect(self.on_click_button_previous_step)
```

Comment:
Here, this is a solution to a bug solved in #60.

However, because we recently merged it you just need to merge your PR with main branch to get the update included in #60.

Reply:
That has  been addressed. The code now works properly, including navigation.
There are no merge conflicts.